client/components/Overview.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import BigRatingStar from './BigRatingStar.jsx';
+
+class Overview extends React.Component {
+  render() {
+    return (
+      <div className="reviews-overview-container">
+        <div className="reviews-overview">
+
+          <p class="heading">Feedback from the Flock</p>
+          <div className="reviews-overview-rating">
+            <div className="rating">
+              <BigRatingStar/>
+              <div className="rating-legend">
+                4.9 out of 5 stars
```

Comment:
Another thing to look at dynamically rendering - I am sure you have a star rating or can generate a star rating out of your db.